### PR TITLE
Adding `paramsKeeper.Subspace(icahosttypes.SubModuleName)`

### PR DESCRIPTION
**Pull request by RaulBernal**
Adding `paramsKeeper.Subspace(icahosttypes.SubModuleName)` at `initParamsKeeper` func

## Description
I think this line is missing in the instructions

### CODE REVIEW

**Comment by RaulBernal** [comment]
BTW, I'm including 2 more checks; I can make a new commit for that if you see it necessary. 

Check2
======
Stargaze include this declaration here:
https://github.com/public-awesome/stargaze/blob/b0bea28fc695a2a5c567e56a37b289a5b75830cc/app/app.go#L714

Gaiad and official instructions to integrate ICA not:
https://github.com/cosmos/gaia/blob/d0884c29aac4c1e647b0a82f3df31515d2bd06a3/app/app.go#L696

Check3
======
Do you think is needed add to Simulation Manager the ICA module?
https://github.com/cosmos/gaia/blob/d0884c29aac4c1e647b0a82f3df31515d2bd06a3/app/app.go#L580
Add: `ica.NewAppModule((*keeper.Keeper)(&app.ICAHostKeeper))`

File: docs/apps/interchain-accounts/integration.md [hunk 1]
```diff
@@ -122,6 +122,12 @@ app.mm.SetOrderInitGenesis(
     icatypes.ModuleName,
     ...
 )
+
+// initParamsKeeper init params keeper and its subspaces
+func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey sdk.StoreKey) paramskeeper.Keeper {
+	...
+	paramsKeeper.Subspace(icahosttypes.SubModuleName)
```
Comment (crodriguezvega) [inline docs/apps/interchain-accounts/integration.md:129]: I see that we are also missing `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)`, so we could also add it.

File: docs/apps/interchain-accounts/integration.md [hunk 2]
```diff
@@ -122,6 +122,12 @@ app.mm.SetOrderInitGenesis(
     icatypes.ModuleName,
     ...
 )
+
+// initParamsKeeper init params keeper and its subspaces
+func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey sdk.StoreKey) paramskeeper.Keeper {
+	...
```
Comment (crodriguezvega) [inline docs/apps/interchain-accounts/integration.md:128]: Just a nit, but do you mind to indent with 4 spaces instead of 8?

**Review by crodriguezvega — CHANGES_REQUESTED** [review]
Thanks for catching this, @RaulBernal!

**Comment by crodriguezvega** [comment]
> # Check2
> Stargaze include this declaration here: https://github.com/public-awesome/stargaze/blob/b0bea28fc695a2a5c567e56a37b289a5b75830cc/app/app.go#L714
> 
> Gaiad and official instructions to integrate ICA not: https://github.com/cosmos/gaia/blob/d0884c29aac4c1e647b0a82f3df31515d2bd06a3/app/app.go#L696

What are the checks that you mention? I think the links above might not be pointing to the right line?

> # Check3
> Do you think is needed add to Simulation Manager the ICA module? https://github.com/cosmos/gaia/blob/d0884c29aac4c1e647b0a82f3df31515d2bd06a3/app/app.go#L580 Add: `ica.NewAppModule(....)`

Yes, I think we can do that now that @colin-axner has merged the PRs for [this issue](https://github.com/cosmos/ibc-go/issues/1352)...

**Review by colin-axner — APPROVED** [review]

**Comment by colin-axner** [comment]
> BTW Should be implemented also in the Simulation test?

Yes now that the ics27 module supports simulation manager usage we should add it to the list of modules

**Comment by crodriguezvega** [comment]
> > BTW Should be implemented also in the Simulation test?
> 
> Yes now that the ics27 module supports simulation manager usage we should add it to the list of modules

Thanks, @colin-axner. I opened this [PR](https://github.com/cosmos/ibc-go/pull/2252). Is that all we need?

File: docs/apps/interchain-accounts/integration.md (hunk 1 above)
Comment (crodriguezvega) [reply docs/apps/interchain-accounts/integration.md]: @RaulBernal do you agree that we should also add this line `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` here for completeness?

**Comment by colin-axner** [comment]
> Thanks, @colin-axner. I opened this [PR](https://github.com/cosmos/ibc-go/pull/2252). Is that all we need?

We can mention it as an optional addition in the docs as well

File: docs/apps/interchain-accounts/integration.md (hunk 1 above)
Comment (crodriguezvega) [reply docs/apps/interchain-accounts/integration.md]: Hey @RaulBernal, you don't agree that we should also add `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` here as well?

Comment (RaulBernal) [reply docs/apps/interchain-accounts/integration.md]: Oh yes sorry

Comment (crodriguezvega) [reply docs/apps/interchain-accounts/integration.md]: Sorry to keep repeating myself :) but could you please add the the line `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` as well if you agree?

Comment (crodriguezvega) [reply docs/apps/interchain-accounts/integration.md]: Sorry to keep repeating myself :) but would you mind to add the line with `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` as well if you agree?

**Review by RaulBernal — COMMENTED** [review]
Added paramsKeeper for ICA Controller

File: docs/apps/interchain-accounts/integration.md (hunk 1 above)
Comment (RaulBernal) [reply docs/apps/interchain-accounts/integration.md]: Done, sorry my delay 🙏

**Review by damiannolan — APPROVED** [review]

**Pull request merged**